packages/tradershub/src/features/cfd/screens/Jurisdiction/JurisdictionCard/JurisdictionCard.tsx

```diff
@@ -0,0 +1,185 @@
+import React, { MouseEvent, useMemo, useState } from 'react';
+import { qtMerge, Text } from '@deriv/quill-design';
+import { useModal } from '../../../../../components/ModalProvider';
```

Comment:
Lets use {Provider} from '@deriv/library'

Reply:
Whats the difference?

Reply:
Will remove provider in tradershub package soon.